app/models/post.rb

```diff
@@ -1,4 +1,8 @@
 class Post < ApplicationRecord
+  validates :title, presence: true, maximum: 250
+  validates :comments_counter, minimum: 0
+  validates :likes_counter, minimum: 0
```

Comment:
- Good job done so far validating your models 👍 👍, but it seems you are using the wrong validation methods. Please check the [ruby guides](https://guides.rubyonrails.org/active_record_validations.html) for the proper validation methods 🙏